packages/generator-langium/templates/packages/extension/package.json

```diff
@@ -28,7 +28,7 @@
     "main": "./out/extension/main.cjs",
     "scripts": {
         "clean": "shx rm -fr *.tsbuildinfo out syntaxes",
-        "vscode:prepublish": "npm run build && npm run lint",
+        "vscode:prepublish": "npm run build && npm run --prefix ../.. lint",
```

Comment:
I can't find any documentation for the `--prefix` option. In fact, there is a closed issue about it at https://github.com/npm/cli/issues/1368#issuecomment-1241076700 that mentions possible deprecation. Perhaps it would be better to use `--workspace`?

Reply:
Thnak you, I didn't know it is a "hidden" feature. I used multiple times before. I adapted the code accordingly. ⬇️